values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/infinispan/server:12.1
+initContainerImage: busybox
+
+replicas: 1
+
+service:
+  type: DataGrid
+  container:
+    storage: 1Gi
+    storageClassName: ""
+    ephemeralStorage: false
+  resources:
+    extraJvmOpts: ""
+    cpu: 500m
+    memory: 512Mi
+
+security:
+  endpointAuthentication: true
+  endpointSecretName: ""
+  endpointSecretPassword: ""
+
+expose:
+  type: Route
+  nodePort: 0
+  host: ""
+  annotations: {}
+
+logging:
+  categories: {}
+
+cloudEvents:
```

Comment:
I'm not sure if `cloudEvents` needs to be this configurable tbh. I think if we move to a model where the user can provide a custom xml, then configuring cloud events via that mechanism should be suffcient. That being said, I'm happy to keep this in for now if you have strong views on this.

Reply:
Yes, lets keep in sync with the most of the operator options